### PR TITLE
Chapter 13 validation

### PR DESCRIPTION
**Pull request by ghost**
- Converted a weird \<figure\> into the usual one with a \<sidebyside\>.
- In sec_triple_int.ptx there is an exercise without a solution given.
- Reorganized figure ids in sec_triple_int.ptx because many ids needed to be added and it (hopefully) makes sense how they look. Original ids preserved in the comments of course.

### CODE REVIEW

File: src_pp/sec_double_int_polar.ptx [hunk 1]
```diff
@@ -1,487 +1,465 @@
...
-      subtended by an angle <m>\theta</m>,
-      is <m>A = \frac12r^2\theta</m>,
-      we can find the area of the shaded region.
-      The whole sector has area <m>\frac12r_2^2\Delta \theta</m>,
-      whereas the smaller,
-      unshaded sector has area <m>\frac12r_1^2\Delta \theta</m>.
-      The area of the shaded region is the difference of these areas:
       <me>
-        \Delta A_i = \frac12r_2^2\Delta\theta-\frac12r_1^2\Delta\theta = \frac12\big(r_2^2-r_1^2\big)\big(\Delta\theta\big) = \frac{r_2+r_1}{2}\big(r_2-r_1\big)\Delta\theta
+        \iint_Rf(x,y)\ dA = \int_\alpha^\beta\int_{g_1(\theta)}^{g_2(\theta)} f\big(r\cos(\theta) ,r\sin(\theta) \big)\ r\ dr\ d\theta
```
Comment (sean-fitzpatrick) [inline src_pp/sec_double_int_polar.ptx]: This doesn't look right. I think git is getting confused by the size of the diff here.
Line numbers are off, and one equation is getting replaced by another.

Comment (sean-fitzpatrick) [reply src_pp/sec_double_int_polar.ptx]: The file looks fine though: everything is in place there.
So maybe it's just not displaying the diff correctly?

File: src_pp/sec_double_int_volume.ptx [hunk 2]
```diff
@@ -305,344 +286,345 @@
...
-        \ds \sum_{i=1}^n f(x_i,y_i)\dx\dy
+        \sum_{j=1}^n\left(\sum_{i=1}^mf(x_i,y_j)\Delta x_i\right)\Delta y_j
       </me>.
     </p>
 
     <p>
-      This approximates the signed volume under <m>f</m> over <m>R</m>.
-      As we have done before,
-      to get a better approximation we can use more rectangles to approximate the region <m>R</m>.
+      The summation inside the parenthesis indicates the sum of heights <times/> widths,
```
Comment (sean-fitzpatrick) [inline src_pp/sec_double_int_volume.ptx]: Same thing here: line numbers are off in the diff. But the file looks fine when viewed.

File: src_pp/sec_surface_area.ptx [hunk 3]
```diff
@@ -257,669 +255,668 @@
...
-      In fact, the tangent plane approximates the function so well that in this figure,
-      it is virtually indistinguishable from the surface itself!
-      Therefore we can approximate the surface area <m>S_i</m> of this region of the surface with the area <m>T_i</m> of the corresponding portion of the tangent plane.
+      The concept of surface area is <em>defined</em> here,
+      for while we already have a notion of the area of a region in the <em>plane</em>,
+      we did not yet have a solid grasp of what
+      <q>the area of a surface in
+      <em>space</em></q> means.
     </p>
+  </aside>
```
Comment (sean-fitzpatrick) [inline src_pp/sec_surface_area.ptx]: things are off here as well

**Review by sean-fitzpatrick — CHANGES_REQUESTED** [review]
Please do a manual diff (e.g. with Meld) for the files with comments (sec_surface_area, sec_double_int_area) between this branch and master.
Just make sure that nothing is getting lost here, since the the diff on GitHub is not lining up correctly.

**Comment by ghost** [comment]
Looks like there were too many sweeping changes from removing comments and removing the \<introduction\> parts which were encompassing the entire section up to the exercises. I added those back in for now and it has greatly simplified the file and no longer looks broken. 

That triple integration file is also very large and might need to be handled the same as space space_coord to get it to look right as well as that file does not handle the introduction section in the same way and the above fix will not apply.

We can create a new PR after this one that removes the <introduction\> in those files which should work as expected then.

**Comment by sean-fitzpatrick** [comment]
Yes, triple integration still has problems.
The diff is showing comparisons that really shouldn't be there, so things are off significantly.

Once there's a reasonable diff for triple integration we can merge this.

**Comment by sean-fitzpatrick** [comment]
Looks like you fixed double_int_polar.
But double_int_volume and surface_area still have bad diffs. Maybe you can try putting the introduction back on those?

**Comment by ghost** [comment]
triple integration still definitely has issues but gits diff is showing the correct changes for both double_int_volume and surface_area. I did add back the introduction for surface_area last night which did fix it for me.

**Comment by ghost** [comment]
Simplified the validation by not deleting comments or tabbing content. Still fixed figure ids and all the normal validation errors. Can fix tabbing when we do the image work.

**Review by sean-fitzpatrick — APPROVED** [review]

**Comment by sean-fitzpatrick** [comment]
This all looks good now. At some point, you'll want to process the whole thing (to HTML, say) to make sure that none of the adjustments to xml:id breaks a reference from another section, but otherwise, no concerns.

**Pull request merged**